runelite-client/src/main/java/net/runelite/client/plugins/discord/DiscordState.java

```diff
@@ -209,4 +211,16 @@ void checkForTimeout()
 		final Instant now = Instant.now();
 		events.removeIf(event -> event.getType().isShouldTimeout() && now.isAfter(event.getUpdated().plus(actionTimeout)));
 	}
-}
+
+	public void checkForMenuTimeout()
+	{
+		EventWithTime event = events.get(0);
```

Comment:
final. Also, you should check if events are not empty first

Reply:
I pushed the fix for it now